configure.ac

```diff
@@ -2862,15 +2895,21 @@ AC_CHECK_HEADERS(linux/qrtr.h,,,[
 #endif
 ])
 
-AC_CHECK_HEADERS(linux/vm_sockets.h,,,[
+AC_CHECK_HEADERS([linux/vm_sockets.h], [], [], [
 #ifdef HAVE_SYS_SOCKET_H
 #include <sys/socket.h>
 #endif
 ])
 
 # On Linux, can.h, can/bcm.h, can/j1939.h, can/raw.h require sys/socket.h
 # On NetBSD, netcan/can.h requires sys/socket.h
-AC_CHECK_HEADERS(linux/can.h linux/can/bcm.h linux/can/j1939.h linux/can/raw.h netcan/can.h,,,[
+AC_CHECK_HEADERS([m4_normalize([
```

Comment:
guessing: Is this perhaps the source of your mystery empty header name `HAVE_` macro in pyconfig.h.in?

missing a trailing \ to escape a newline or otherwise should just leave these all on one line w/o a m4_normalize perhaps?

Reply:
But no, it is not. The dog is buried somewhere else.

Reply:
Yes, _this is the culprit_. Thanks!

Reply:
Resolved with 68bc05e5a63eabf166bec3e612a5ff624038b408